src/lib/WaveIcon.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+ import type { WaveType } from '$lib/audio/instrument';
+ export let wave: WaveType;
+ export let selected: boolean;
+</script>
+
+<svg
+    width="200mm"
+    height="200mm"
+    viewBox="-30 -30 260 260"
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+    preserveAspectRatio="none"
+>
+    <path class="waveIcon {wave}" class:selected d="m 0,100 V 0 H 100 V 200 H 200 V 10"/>
```

Comment:
waveIcon class is pointless here because you're not using it in CSS. Either:
- remove it here and rely on svelte style scoping, or
- use it in CSS like .waveIcon.sine {...}